charts/sda-svc/templates/auth-deploy.yaml

```
@@ -135,6 +135,8 @@ spec:
           value: {{ .Values.global.auth.jwtAlg }}
         - name: RESIGNJWT
           value: {{ .Values.global.auth.resignJwt | quote }}
+        - name: PUBLICFILE
+          value: "/keys/c4gh.pub.pem"
```

```suggestion
          value: "{{ template "c4ghPath" . }}/{{ .Values.global.c4gh.publicFile }}"
```

Never hard code stuff that are variables defined in the values file.

---

This was me trying to debug the failing job, probably your next comment is the reason.